test/ssl-test_test.rb

```diff
@@ -90,7 +90,7 @@
 
     it "stops following redirection after the limit for the revoked certs check" do
       valid, error, cert = SSLTest.test("https://github.com/", redirection_limit: 0)
-      error.must_equal "OCSP test couldn't be performed: OCSP response request failed"
+      error.must_equal "OCSP test couldn't be performed: OCSP response request failed (OCSP URI: http://ocsp.digicert.com, error: Too many redirections (> 0))"
```

Comment:
That's a lot of "OCSP" :) we can probably shorten this a bit, like:
`OCSP test couldn't be performed: request failed (URI: http://ocsp.digicert.com, error: Too many redirections (> 0))`
?

Reply:
Ok, done!

```ruby
irb(main):001:0> SSLTest.test("https://cloudreports.xyz")
=> [true, "OCSP test couldn't be performed: request failed (URI: http://isrg.trustid.ocsp.identrust.com, error: Wrong response type (Net::HTTPServiceUnavailable))", #<OpenSSL::X509::Certificate: subject=#<OpenSSL::X509::Name CN=cloudreports.xyz>, issuer=#<OpenSSL::X509::Name CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US>, serial=#<OpenSSL::BN:0x0000000007b745d8>, not_before="2020-03-10T03:25:33.000Z", not_after="2020-06-08T03:25:33.000Z">]
```

Reply:
Thanks, I updated a couple other error messages to reduce a bit the duplication, let me know if you have any feedback on that.

Reply:
Nothing to add.